apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.GenerateOverloads/Methods/TableCrud.xml

```diff
@@ -29,7 +31,11 @@
       TargetType="Table"
       ReturnType="BigQueryTable">
     <AdditionalParameters>
-      <Parameter Name="schema" Type="TableSchema" Comment="The schema to use to create the table if necessary. Must not be null unless the schema can be inferred (e.g. for a view)." />
+      <Parameter Type="GetTableOptions" Name="getOptions" Comment="The options for the &quot;get&quot; operation. May be null, in which case defaults will be supplied." DefaultValue="null" />
```

Comment:
Now that these are not optional (schema or table) they have to come before the options anyway. Won't break existing users of the schema overload because schema wasn't optional before and came before the options (although it did allow nulls).